optaplanner-docs/src/main/asciidoc/RepeatedPlanning/continuousPlanningPatientAdmissionSchedule.svg

```diff
@@ -2,32 +2,31 @@
 <!-- Created with Inkscape (http://www.inkscape.org/) -->
 
 <svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:xlink="http://www.w3.org/1999/xlink"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="800"
-   height="600"
-   id="svg2"
-   version="1.1"
-   inkscape:version="0.92.4 (unknown)"
-   sodipodi:docname="continuousPlanningPatientAdmissionSchedule.svg"
-   inkscape:export-filename="continuousPlanningPatientAdmissionSchedule.png"
-   inkscape:export-xdpi="90"
-   inkscape:export-ydpi="90">
+    xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:cc="http://creativecommons.org/ns#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+    width="800"
+    height="600"
+    id="svg2"
+    version="1.1"
+    inkscape:version="0.92.4 (unknown)"
+    sodipodi:docname="continuousPlanningPatientAdmissionSchedule.svg"
+    inkscape:export-filename="continuousPlanningPatientAdmissionSchedule.png"
+    inkscape:export-xdpi="90"
+    inkscape:export-ydpi="90">
```

Comment:
(not for this PR) dpi is 96 these days - I 'll explain in one of our meetings, but don't worry about it for this PR.